.goreleaser.yml

```diff
@@ -29,7 +29,7 @@ builds:
 nfpms:
   - id: packages
     package_name: infra
-    file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Arch }}"
+    file_name_template: "{{ .ConventionalFileName }}"
```

Comment:
goreleaser has some reasonable defaults for the different distros so we'll use that instead of defining overrides for everything